src/main/java/mesosphere/marathon/metrics/Metrics.scala

```diff
@@ -65,6 +66,16 @@ class Metrics @Inject() (val registry: MetricRegistry) {
 
 object Metrics {
   class Timer(timer: com.codahale.metrics.Timer) {
+    def future[T](future: => Future[T]): Future[T] = {
```

Comment:
Please rename to something like timed(future).
In the code it looks like xxx.future( {some action} ) and my first impression was, a future is created...